portal_pages/tests/test_user_forms.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+
+from django.test import TestCase
+
+from wagtail.wagtailcore.models import Page, Site
+
+from portal_pages.models import (
+    MarketplaceIndexPage, MarketplaceEntryPage,
+    CaseStudyIndexPage, CaseStudyPage,
+    BlogIndexPage, BlogPage
+    )
+
+
+class MarketplaceEntryPageAddTests(TestCase):
+
+    def setUp(self):
+        # Wagtail's default site comes with a Root (1) and Home Page (2)
+        self.default_site = Site.objects.get(is_default_site=True)        
+        self.root = Page.objects.get(pk=1)
+        self.home_page = Page.objects.get(pk=2)
+
+        # Add a marketplace index page under the home page
+        self.marketplace_index_page = self.home_page.add_child(
+            instance=MarketplaceIndexPage(
+                title="Marketplace",
+                slug="marketplace",
+                live=True,
+            ))
+        self.marketplace_submission_slug = '/%s/submit-marketplace-entry/' % self.marketplace_index_page.slug
```

Comment:
I knew there was some way to do this in django, but wasn't sure what to pass in to reverse_subpage(), thank you.